dataflow/custom-containers/minimal/Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM python:3.8-slim
+
+COPY requirements.txt .
+COPY main.py ./
+
+# Install the requirements.
+RUN pip install --no-cache-dir -r requirements.txt \
```

Comment:
nit: the requirements should install before the `COPY main.py ./`. This way the requirements are cached if the `requirements.txt` hasn't changed but main.py has

Reply:
Thanks, rearranged copy/install requirements first, then copy source files.